tests/misc/src/exprs.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
+
+int side_effect(int* x) {
+        *x = 1;
+        return 0;
+}
+
+void exprs(const unsigned n, int * const buffer) {
+
+        unsigned long i = 0;
+		char arr[1] = {0};
+
+        side_effect(&buffer[0]);
+        -side_effect(&buffer[1]);
+        &""[side_effect(&buffer[2])];
+		++arr[side_effect(&buffer[3])];
+}
```

Comment:
nit: Indentation.

Reply:
I meant the 8-space indentation.

Reply:
I was following the pattern of `sizeofs.c`. We should normalize all these, but I didn't do a survey to see what was most common.

Reply:
Ahh, okay.